src/applications/mhv-secure-messaging/tests/e2e/keyboard-nav-test/secure-messaging-keyboard-nav-to-links-buttons-landing-page.cypress.spec.js

```diff
@@ -20,15 +20,19 @@ describe('Secure Messaging Verify Links and Buttons Keyboard Nav', () => {
       .find('a')
       .should('have.focus');
 
-    cy.realPress('Tab');
     cy.get(Locators.ALERTS.ACC_ITEM).each(el => {
-      cy.get(el).should('have.focus');
       cy.realPress('Tab');
+      cy.get(el).should('have.focus');
     });
 
+    cy.get(Locators.ALERTS.BACK_TOP).scrollIntoView();
+
     cy.get(Locators.ALERTS.BACK_TOP)
-      .scrollIntoView()
-      .should('have.focus');
+      .shadow()
+      .find('.docked.reveal', { timeout: 1000 })
+      .should('be.visible');
+    cy.realPress('Tab');
+    cy.get(Locators.ALERTS.BACK_TOP).should('have.focus');
```

Comment:
I have slightly tweaked this test to maintain validation of focus. I found that cypress will treat this component as visible although it is not, which will make `be.visible` validation false positive.